nototools/noto_lint.py

```diff
@@ -362,7 +362,9 @@ def printable_font_versions(font):
 
 
 def _build_cmap_dict(filename):
-    data = cmap_data.read_cmap_data_file(filename)
+    tooldir = tool_utils.resolve_path('[tools]/nototools')
+    data = cmap_data.read_cmap_data_file(
+        path.join(tooldir, filepath))
```

Comment:
This doesn't come up that much, so it doesn't seem worth it to me.  I could have modified the file names to include the complete path too, which would also have worked.